src/debugger/rnDebugSession.ts

```diff
@@ -91,48 +92,57 @@ export class RNDebugSession extends LoggingDebugSession {
                             extProps = TelemetryHelper.addPropertyToTelemetryProperties(versions.reactNativeWindowsVersion, "reactNativeWindowsVersion", extProps);
                         }
                         return TelemetryHelper.generate("attach", extProps, (generator) => {
-                            logger.log(localize("StartingDebuggerAppWorker", "Starting debugger app worker."));
-
-                            const sourcesStoragePath = path.join(this.projectRootPath, ".vscode", ".react");
-                            // Create folder if not exist to avoid problems if
-                            // RN project root is not a ${workspaceFolder}
-                            mkdirp.sync(sourcesStoragePath);
-
-                            // If launch is invoked first time, appWorker is undefined, so create it here
-                            this.appWorker = new MultipleLifetimesAppWorker(
-                                attachArgs,
-                                sourcesStoragePath,
-                                this.projectRootPath,
-                                undefined);
-
-                            this.appWorker.on("connected", (port: number) => {
-                                logger.log(localize("DebuggerWorkerLoadedRuntimeOnPort", "Debugger worker loaded runtime on port {0}", port));
-
-                                const attachArguments = {
-                                    type: "node",
-                                    request: "attach",
-                                    name: "Attach",
-                                    port: port,
-                                };
-
-                                vscode.debug.startDebugging(
-                                    this.appLauncher.getWorkspaceFolder(),
-                                    attachArguments,
-                                    this.session
-                                )
-                                .then((childDebugSessionStarted: boolean) => {
-                                    if (childDebugSessionStarted) {
-                                        resolve();
-                                    } else {
-                                        reject(new Error("Can not start child debug session"));
-                                    }
-                                },
-                                err => {
-                                    reject(err);
+                            this.rnCdpProxy = new RnCdpProxy();
```

Comment:
Why we don't initialize CDP Proxy with target URL host and port?

Reply:
We can provide host URL and a port while initialization. For now default parameters in CDP Proxy class constructor are used. I'll add concrete parameters to Proxy constructor call.